src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/MsftToSbSdk.diff

```diff
@@ -545,4 +545,6 @@ index ------------
 -./sdk/x.y.z/zh-Hant/Microsoft.VisualStudio.Coverage.IO.resources.dll
  ./sdk/x.y.z/zh-Hant/MSBuild.resources.dll
  ./sdk/x.y.z/zh-Hant/System.CommandLine.resources.dll
- ./shared/
\ No newline at end of file
+ ./sdk/x.y.z/zh-Hant/vstest.console.resources.dll
+ ./shared/
+ ./shared/Microsoft.AspNetCore.App/x.y.z/
```

Comment:
Looks like you dropped the extra empty line.  IIRC this will cause the diff/test to fail.

Reply:
Ah, trusted the Git highlighting too much.  Fixed, thanks.